### ISSUE TITLE
`getDefaultDocumentNode` in deskStructure seems not to get picked up

### ISSUE THREAD

**Issue by mordaroso** [sanity-io/document-internationalization#24]
**Version:** `0.0.1-beta.8`

I would like to use [Live Preview](https://www.sanity.io/guides/nextjs-live-preview) and [document-internationalization](https://github.com/sanity-io/document-internationalization) and have them both as document views:
<img width="217" alt="Screenshot 2022-02-17 at 14 29 57" src="https://user-images.githubusercontent.com/15565/154491641-8d0486f6-810c-42d1-a455-cff5355d123c.png">

Note: This works well with the previous [sanity-plugin-intl-input](https://github.com/LiamMartens/sanity-plugin-intl-input) plugin. So I assume it's a regression.

The [setup documentation for Desk structure](https://github.com/sanity-io/document-internationalization/blob/main/docs/desk-structure.md) explains how to use `getDefaultDocumentNode` but using `Structure.getFilteredDocumentTypeListItems()` seems to skip the method entirely for any document type with `i18n: true`.

Example:
```js
import * as Structure from '@sanity/document-internationalization/lib/structure'
import S from '@sanity/desk-tool/structure-builder'

export const getDefaultDocumentNode = (props) => {
  throw new Error(props.schemaType); // <= This will never execute for schemas with I18n therefore not throw any error
}

export default () => {
  const items = Structure.getFilteredDocumentTypeListItems()
  return S.list().id('__root__').title('Content').items(items)
}
```

It works when the default from structure-builder is used:
```js
import S from '@sanity/desk-tool/structure-builder'

export const getDefaultDocumentNode = (props) => {
  throw new Error(props.schemaType); // <= This will throw an error
}

export default () => S.defaults()
```

Going through the source code I suspect it's the change in [`getFilteredDocumentTypeListItems`](https://github.com/sanity-io/document-internationalization/blob/main/src/structure/index.tsx#L95-L97) that overwrites the views.

**Comment by LiamMartens** [comment]
@mordaroso this will be fixed in [#26](https://github.com/sanity-io/document-internationalization/pull/26) as the new UI will not require `getDefaultDocumentNode` any longer. I'll close this issue once released.

**Comment by oliverdunk** [comment]
Is this definitely resolved? I think it might still be a problem because `getFilteredDocumentTypeListItems` is coming back with a custom child resolver. I had to override that to get things working on my end.

**Comment by LiamMartens** [comment]
@oliverdunk you were right! This is now fully resolved in `0.1.5`
